### PR TITLE
[SNOW-60] Hotfix: Programmatically feed stage name

### PR DESCRIPTION
**Pull request by jaymedina**
Instead of programmatically feeding the stage name, I had it hardcoded to `prod`, which broke when deploying to `dev`.

This PR patches that up in both the task and refresh scripts.

### CODE REVIEW

**Review by philerooski — APPROVED** [review]

File: synapse_data_warehouse/synapse_raw/tables/V2.28.1__update_userprofilesnapshots_task.sql
Comment (thomasyu888) [inline synapse_data_warehouse/synapse_raw/tables/V2.28.1__update_userprofilesnapshots_task.sql:40]: Do not merge: 

```
        FROM @{{stage_storage_integration}}_stage/nodesnapshots/ --noqa: TMP
```

**Review by thomasyu888 — COMMENTED** [review]
🔥 LGTM! Sorry about the nuance around the stage name env variable. I defer to @philerooski for final re-review.

**Review by philerooski — APPROVED** [review]
Huh I thought I already approved the hotfix.

**Comment by jaymedina** [comment]
I had to add the `_stage` part

**Comment by philerooski** [comment]
@jaymedina ah, okay. For very minor changes, if I've already approved then there's no need to re-request. I don't want to be a blocker for typo fixes etc.

**Comment by thomasyu888** [comment]
@philerooski, for transparency, I had requested for your final re-review, as I noticed that typo and made a comment.  Another thing I noticed was that the COPY into script isn't using the COMPUTE_MEDIUM warehouse, so copying into for the production warehouse is going to take that much longer if it took 15 minutes to copy ~400k rows.

**Comment by jaymedina** [comment]
> Another thing I noticed was that the COPY into script isn't using the COMPUTE_MEDIUM warehouse, so copying into for the production warehouse is going to take that much longer if it took 15 minutes to copy ~400k rows.

I see - then I think it's worth establishing a set of guidelines somewhere for what warehouse to use for a given script (e.g. if you are loading >500k rows worth of data, use MEDIUM)

We can use the docs [here](https://docs.snowflake.com/en/user-guide/warehouses-overview#impact-on-data-loading) for reference

In the meantime, I'll make another PR to update to MEDIUM

**Pull request merged**
